synapse/handlers/e2e_keys.py

```diff
@@ -259,34 +292,60 @@ def claim_client_keys(destination):
             ),
         )
 
+        opentracing.log_kv({"one_time_keys": json_result, "failures": failures})
         return {"one_time_keys": json_result, "failures": failures}
 
     @defer.inlineCallbacks
+    @opentracing.tag_args
```

Comment:
is this the only place this decorator is used? wouldn't it be easier to inline it?

Reply:
It would be easier to inline it here but 'opentracing.py' was written with the view that other people could easily use it and I think this is a very handy decorator. Especially during debug!